pkg/image/bootc_disk.go

```diff
@@ -89,5 +91,17 @@ func (img *BootcDiskImage) InstantiateManifestFromContainers(m *manifest.Manifes
 		fmt.Sprintf("%s.vmdk", fileBasename),
 		fmt.Sprintf("%s.vhd", fileBasename),
 	}
+
+	// XXX: copied from https://github.com/osbuild/images/blob/v0.85.0/pkg/image/disk.go#L102
+	gcePipeline := manifest.NewTar(buildPipeline, rawImage, "gce")
+	gcePipeline.Format = osbuild.TarArchiveFormatOldgnu
+	gcePipeline.RootNode = osbuild.TarRootNodeOmit
+	// these are required to successfully import the image to GCP
+	gcePipeline.ACLs = common.ToPtr(false)
+	gcePipeline.SELinux = common.ToPtr(false)
+	gcePipeline.Xattrs = common.ToPtr(false)
+	gcePipeline.Transform = fmt.Sprintf(`s/%s/disk.raw`, regexp.QuoteMeta(rawImage.Filename()))
```

Comment:
Manual test indicates this Quote is not quite matching what we need which is a bit sad, but from a manual run:
``` 
Build
  root: 2d64064ce5eda7ccf36bac182781df1683350c22a9db6da06c4b057f9d96e89e
  runner: org.osbuild.linux (org.osbuild.linux)
org.osbuild.tar: c46ccd654e6c9a746915de901da107aff3d9302d44a1cd9105b660c0a07c0e90 {
  "filename": "image.tgz",
  "format": "oldgnu",
  "acls": false,
  "selinux": false,
  "xattrs": false,
  "root-node": "omit",
  "transform": "s/disk\\.raw/disk.raw"
}
tar: Invalid transform expression
Try 'tar --help' or 'tar --usage' for more information.
Traceback (most recent call last):
  File "/run/osbuild/bin/org.osbuild.tar", line 74, in <module>
    r = main(args["inputs"], args["tree"], args["options"])
  File "/run/osbuild/bin/org.osbuild.tar", line 62, in main
    subprocess.run(
  File "/usr/lib64/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['tar', '--auto-compress', '--format=oldgnu', '--transform', 's/disk\\.raw/disk.raw', '-cf', '/run/osbuild/tree/image.tgz', '-C', '/run/osbuild/inputs/tree', 'disk.raw']' returned non-zero exit status 2.

⏱  Duration: 0s
manifest /tmp/lala.json failed
Failed

```